qa/1599

```diff
@@ -28,41 +29,18 @@ status=1	# failure is the default!
 $sudo rm -rf $tmp $tmp.* $seq.full
 trap "_cleanup; exit \$status" 0 1 2 3 15
 
-pmdastatsd_remove()
-{
-    echo
-    echo "=== remove statsd agent ==="
-    cd $PCP_PMDAS_DIR/statsd
-    $sudo ./Remove >$tmp.out 2>&1
-    _filter_pmda_remove <$tmp.out
-}
-
-pmdastatsd_install()
-{
-    echo
-    echo "=== statsd agent installation ==="
-    cd $PCP_PMDAS_DIR/statsd
-    $sudo ./Install </dev/null >$tmp.out 2>&1
-    cat $tmp.out >>$here/$seq.full
-    _filter_pmda_install <$tmp.out
-}
-
 _prepare_pmda statsd
 # note: _restore_auto_restart pmcd done in _cleanup_pmda()
 trap "_cleanup_pmda statsd; exit \$status" 0 1 2 3 15
 _stop_auto_restart pmcd
 
-# real QA test starts here
-pmdastatsd_install
-
-cd $here/statsd
-scripts=`ls *.rb | LC_COLLATE=POSIX sort`
+cd $here/statsd/src
+scripts=`find cases -name *.py | LC_COLLATE=POSIX sort`
 for script in $scripts
 do
-    ruby $script
+    python $script $here/statsd/output
```

Comment:
Using common.python, this should then be $python rather than just python.

This lets us test both python2 and python3 more easily (we use python3 more aggressively - whenever available - over python2).